react-native/react/engine/index.js

```diff
@@ -73,7 +69,43 @@ class Engine {
     const callMap = this.sessionIDToIncomingCall[sessionID]
 
     if (callMap && callMap[method]) {
-      callMap[method](param, response)
+      // make wrapper so we only call this once
+      let once = false
```

Comment:
I needed some help debugging this whole thing so i added a flag to local-debug to print rpc stuff in chrome